Tests/Segment-Tests/Support/TestUtilities.swift

```diff
@@ -107,3 +107,16 @@ class OutputReaderPlugin: Plugin {
         return event
     }
 }
+
+func waitUntilStarted(analytics: Analytics?) {
+    guard let analytics = analytics else { return }
+    var started = false
+    while started == false {
+        if let system: System = analytics.store.currentState() {
```

Comment:
Wouldn't mind some commented context =). Makes sense right now in the moment, but what about three months from now?

Reply:
nevermind, just noticed this is in tests